src/mixins/filing-template-mixin.ts

```diff
@@ -304,6 +304,7 @@ export default class FilingTemplateMixin extends DateMixin {
         // add in the fields needed by Legal API
         ...this.getNameRequest,
         legalName: this.getNameRequestLegalName || undefined, // don't include if empty
+        legalType: this.getEntityType,
         nrNumber: this.getNameRequestNumber
```

Comment:
![image](https://github.com/user-attachments/assets/549473e8-cff8-45e4-9482-4af97eabf6f9)
Per Vysakh's notes, I updated the lteration.nameRequest.legalType to be synced with the altered legal type

Reply:
But for the first issue, I tested with other businesses, the alterations are not synced to colin either. I'm not familiar with this, may I know the possible reasons?

Reply:
Vysakh identified that the sync didn't work for some reason. I just wanted him to know about it. That's outside the scope of this ticket so don't worry about it.